src/main/java/command/CommandType.java

```diff
@@ -0,0 +1,58 @@
+package command;
+
+import java.util.HashMap;
+// 1. command type
+// 1.1 list
+// 1.2 add
+// 1.3 delete
+// 1.3 find
+// 2. action
+// 3. time
+// 3.1 todo
+// 3.2 deadline
+// 3.3 event
```

Comment:
This comment doesnt really explain what you are trying to do and furthermore since you are using an `enum` as a read it is quite clear to me what you are trying to do